pythonFiles/include/blender_vscode/utils_files.py

```diff
@@ -0,0 +1,50 @@
+import os
+import subprocess
+import sys
+from typing import Optional, Union
+
+
+def resolve_link(path: Union[str, os.PathLike]) -> Optional[str]:
```

Comment:
handling juction is an absolute disaster due to python changes 3.7 -> 3.11. In worst case scenario use cmd call.